certified-connectors/Cascade Strategy/apiDefinition.swagger.json

```diff
@@ -310,20 +257,77 @@
             }
           }
         },
-        "summary": "Update Measure Historical Value",
-        "description": "Updates historical values of the selected measure.",
-        "x-ms-visibility": "important",
-        "operationId": "UpdateMeasureHistoricalValue",
-        "consumes": ["application/json"],
+        "summary": "Update Measure",
+        "operationId": "UpdateMeasure",
```

Comment:
The path template has been changed for the operation. ValuePath: operation/UpdateMeasure, can you please confirm that no existing flows or users will be affected by this change?

Reply:
I think this one got mixed up because I changed the order of the paths